src/SearchBox.jsx

```diff
@@ -76,18 +78,47 @@ var HitList = React.createClass({
 
 var SearchBox = React.createClass({
 
+    componentDidMount: function () {
+        document.addEventListener('mousedown', this.handleClickOutside);
+    },
+
+    componentWillUnmount: function () {
+        document.removeEventListener('mousedown', this.handleClickOutside);
+    },
+
     getDefaultProps: function () {
-        return {placeholder: 'Søk', closeOnSelect: true};
+        return {
+            placeholder: 'Søk',
+            closeOnSelect: true,
+            targets: ['matrikkelenhet', 'gateadresse']
+        };
     },
 
     getInitialState: function () {
-        return {text: '', hits: [], hoverIndex: null, selectedIndex: null};
+        return {
+            text: '',
+            hits: [],
+            hoverIndex: null,
+            selectedIndex: null,
+            resultStatus: 'ok',
+            displayHits: 'none'
```

Comment:
why is this the string "none"?

Reply:
Ups, just forgot to change it. Thanks :)